modeltrans/static/modeltrans/css/i18n_tabs.css

```diff
@@ -0,0 +1,40 @@
+.i18n-tabs {}
+
+.i18n-tab-buttons {
+    display: flex;
+    justify-content: start;
+    gap: 10px;
```

Comment:
Ik think this looks a bit more like tabs:


```suggestion
    gap: 5px;
```

![Screenshot 2024-07-02 at 10 40 07](https://github.com/zostera/django-modeltrans/assets/1470389/e2b0748a-685a-41d5-81d8-19d85d097295)

Reply:
It would be nice if the admin views in the example contain at least one view using the `TabbedLanguageMixin`, to make it easy to verify it still works.

Reply:
Serves me right for blindly force-pushing... I added a test and an example Admin class but it got lost in my rebases. 😅